Core/PSPLoaders.cpp

```diff
@@ -322,5 +323,14 @@ bool Load_PSP_ELF_PBP(FileLoader *fileLoader, std::string *error_string) {
 	pspFileSystem.Mount("umd0:", fs);
 
 	std::string finalName = ms_path + file + extension;
+
+	std::string homebrewName = PSP_CoreParameter().fileToStart;
```

Comment:
Demos in pbp format doesn't use that code, PBP for homebrew is basically elf + param.sfo + some icons in it's own directory, demo's of commercial games doesn't differ from full games in PBP format and have same structure as iso.